tests/unit_tests/CMakeLists.txt

```diff
@@ -6,10 +6,13 @@
 
 # Set up all of the "host" tests.
 add_subdirectory( core )
-add_subdirectory( io )
 add_subdirectory ( cpu )
 add_subdirectory( covfie )
 
+if( DETRAY_IO )
```

Comment:
Okay, apparently the variable is used in a second place as well... Still, I don't believe it is warranted...